docs/DeveloperGuide.md

```diff
@@ -536,50 +536,52 @@ This is a quick overview of all the commands and their functionalities.
-| `* `     | hotel manager            | get the details of where the guests booked their hotel stay from   | decide which booking platform I need to pay more attention to                             |
-| `* `     | hotel manager            | get the details of how many guests are eating breakfast            | prepare the appropriate amount of food                                                    |
-| `* `     | hotel manager            | get the details of how many guests are eating lunch                | prepare the appropriate amount of food                                                    |
-| `* `     | hotel manager            | get the details of how many guests are eating dinner               | prepare the appropriate amount of food                                                    |
+| Priority | As a…                    | I want to…                                                         | So that I can…                                                                               |
+|----------|--------------------------|--------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| `* * *`  | first time hotel manager | know the available commands                                        | use the application to its full capabilities                                                 |
+| `* * *`  | hotel manager            | view a list of all the guests                                      | browse all my guests' data at once                                                           |
+| `* * *`  | hotel manager            | view the contact number of the guest                               | contact them in an emergency                                                                 |
+| `* * *`  | hotel manager            | view the name of the guest                                         | verify the guest                                                                             |
+| `* * *`  | hotel manager            | view the email address of the guest                                | contact the guest via email                                                                  |
+| `* * *`  | hotel manager            | be able to exit the program                                        | close it when I am done using it                                                             |
+| `* * *`  | first time hotel manager | be able to delete data of all guests at one go                     | have a clean slate to work on should I need to                                               |
+| `* * *`  | hotel manager            | view the check-in date of a guest                                  | calculate the number of days resided by the guest (using check-out date information as well) |
```

Comment:
I think calculate the number of days the guest stays in the doesn't seem very useful for a user? Should we change this to something else? I think the previous use cases was to prepare the room, can I check why you decided to change that one?

Reply:
I changed that to be more aligned with the fact that we are checking in the guest when the guest arrives, so it wouldn't really be possible to prepare the room in advance since the guest would already be there, but perhaps we can keep it the same then since we can still prepare the room!

Reply:
Ahh I get what you mean!